### DISCUSSION TITLE
iOS widget reading time

### DISCUSSION THREAD

**Discussion by yoflags** [LeoKlaus/plappa#173] [Feature Requests]
I would love it if you could add iOS widget. I saw another discussing where they asked to add a "continue reading" widget. Though this is a good idea I would like a widget that tells me the time reading so I can track habits and it motivates me to read more. Just like the bookplayer app.
![IMG_2931](https://github.com/user-attachments/assets/8f879f9b-9a3d-4715-8884-60329a6c60d6)

**Comment by LeoKlaus** [comment]
This is a cool idea and all of the necessary information should be tracked now, I'll definitely consider this.
